internal/distro/rhel84/distro_v2.go

```diff
@@ -127,7 +127,7 @@ func (t *imageTypeS2) Manifest(c *blueprint.Customizations,
 	}
 
 	var commits []ostreeCommit
-	if options.OSTree.Parent != "" && options.OSTree.URL != "" {
+	if t.bootISO && options.OSTree.Parent != "" && options.OSTree.URL != "" {
```

Comment:
Even though changing old distro definitions should be a no-no, and also should have no effect as we don't plan to do any more 8.4 releases, I think this is ok, as it only avoids a download but otherwise should have no effect. Want to point out that in general we should avoid it though.